Nc-News/src/components/Containers/ArticleCard.jsx

```diff
@@ -0,0 +1,19 @@
+export function ArticleCard({ article }) {
+  const created = new Date(article.created_at);
+  return (
+    <li className="articleCard">
```

Comment:
I would argue that the li doesn't need to be here, and if you wish to keep it here there is no need for a div as the li is in essence a box anyway, you have a box with a title, then another box.

Reply:
Also, don't forget your li key